build/build-image/Dockerfile

```diff
@@ -25,7 +25,8 @@ RUN chmod -R a+rwx /usr/local/go/pkg
 # of operations.
 ENV HOME /go/src/k8s.io/kubernetes
 WORKDIR ${HOME}
-RUN chmod -R a+rwx ${HOME}
+RUN mkdir -p ${HOME} ${HOME}/_output \
```

Comment:
This is super strange. But I agree it doesn't make sense to spend more time on it.